app/templates/index.hbs

```diff
@@ -1,67 +1,47 @@
+      alt="Foto van een laptop met daarop het vlaanderen logo."
+    />
+  </AuContentHeader>
   <AuBodyContainer class="au-u-background-gray-100">
     <section class="c-home">
       <div class="au-o-flow">
         <AuContent>
-          <AuHeading @level="1" @skin="2">Welkom bij Hackathon!</AuHeading>
-        </AuContent>
-        <div class="au-c-home__form au-o-flow">
-          <div class="au-o-grid au-o-grid--small">
-            <div class="au-o-grid__item au-u-1-2@medium">
-              <AuCard @flex={{true}} as |c|>
-                <c.header
-                  @badgeSkin="brand"
-                  @badgeIcon="sitemap"
-                  @badgeSize="small"
-                >
-                  <AuHeading @level="2" @skin="4">
-                    First Card
-                  </AuHeading>
-                </c.header>
-                <c.content>
-                  <AuList @divider={{true}} as |Item|>
-                    <Item>Item #1</Item>
-                    <Item>Item #2</Item>
-                  </AuList>
-                </c.content>
-                <c.footer>
-                  <div class="au-u-h-functional au-o-flow au-o-flow--small">
-                    <AuLink @skin="button" @route="index">Link #1</AuLink>
-                    <AuLink @skin="button-secondary" @route="index">Link #2</AuLink>
-                  </div>
-                </c.footer>
-              </AuCard>
-            </div>
-            {{#if this.currentSession.canEdit}}
+          <div class="au-c-home__form au-o-flow">
+            <div class="au-o-grid au-o-grid--small">
               <div class="au-o-grid__item au-u-1-2@medium">
-                <AuCard @flex={{true}} as |c|>
-                  <c.header
-                    @badgeSkin="brand"
-                    @badgeIcon="upload"
-                    @badgeSize="small"
-                  >
-                    <AuHeading @level="2" @skin="4">
-                      Authorized access
-                    </AuHeading>
-                  </c.header>
-                  <c.content>
-                    <AuList @divider={{true}} as |Item|>
-                      <Item>Item #1</Item>
-                      <Item>Item #2</Item>
-                    </AuList>
-                  </c.content>
-                  <c.footer>
-                    <div class="au-u-h-functional au-o-flow au-o-flow--small">
-                      <AuLink @skin="button" @route="index">Link #1</AuLink>
-                    </div>
-                  </c.footer>
-                </AuCard>
+                <img
+                  src="/assets/images/undraw_house_searching_re_stk8.svg"
+                  alt="oeps"
+                />
               </div>
-            {{/if}}
-          </div>
+              <div class="au-o-grid__item au-u-1-2@medium">
+                <AuHeading @level="2" @skin="4" class="au-u-margin-bottom">
```

Comment:
I used au-u-margin  for the spacing, you could also do this with padding if you want